source/agora/node/Ledger.d

```diff
@@ -25,17 +26,18 @@ import agora.node.API;
 import vibe.core.log;
 
 import std.algorithm;
+import std.file;
+import std.path;
 
 /// Ditto
 public class Ledger
 {
-    /// data storage for all the blocks,
+    /// Blockstorage for all the blocks,
     /// currently a single contiguous region to
-    /// improve locality of reference
-    private const(Block)[] ledger;
+    private BlockStorage ledger;
```

Comment:
I don't like calling this object instance `ledger`. The `Ledger` class is already the ledger, this should be named `storage` or `block_storage` or something else.

Reply:
change name `storage`